hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -705,6 +762,7 @@ private static void printUsage(final String errorMsg) {
     }
     System.err.println("Usage: verifyrep [--starttime=X]"
       + " [--endtime=Y] [--families=A] [--row-prefixes=B] [--delimiter=] [--recomparesleep=] "
```

Comment:
i dont love how this is recomparesleep, but your new options are reCompareFoo. It would be nice to be consistent, and your new ones are easier to read. We can't simply change the job to only accept reCompareSleep, but we can update the usage here and update the parsing to support both (so accept recomparesleep or reCompareSleep)

when parsing recomparesleep, i would add a LOG.warn that it is deprecated and will be removed in 4.0.0, prefer reCompareSleep.

finally, this is a nitpick but `recompare` is a word on its own... so imo it should be `recompareThreads`, `recompareTries`, etc... i.e. don't capitalize the C.

so in summary:

- update these new args to be `recompareFoo` rather than `reCompareFoo`
- update the usage line here to be `recompareSleep` rather than `recomparesleep`
- update arg the parsing above to support both, but print a LOG.warn if the old one is used
- the warn should say "--recomparesleep is deprecated and will be removed in 4.0.0. Use --recompareSleep instead."